.github/workflows/release.yaml

```diff
@@ -5,18 +5,25 @@ on:  # yamllint disable-line rule:truthy
   workflow_call:
   workflow_dispatch:
     inputs:
-      version:
-        description: 'Version to release (major, minor, patch), Example: 1.x.x'
-        required: true
-      image:
-        description: 'Image name. Example: csi-powermax'
-        default: 'csi-powermax'
+      option:
+        description: 'Select version to release'
         required: true
+        type: choice
+        default: 'minor'
+        options:
+          - major
+          - minor
+          - patch
+      reverseproxyCheckInput:
+        description: 'Release CSI-ReverseProxy Image'
+        type: boolean
+        default: false
+        required: false
```

Comment:
Should this be made as 'required' as we always release both the images?

Reply:
The requirement may be that it is possible to release only one image in the case of a patch.

Reply:
There may be an instance where no changes were made to the reverse proxy so it is advised by POs not to release it. That's why driver and reverse proxy are not at the same tag right now. I think its ok to not require reverse proxy on release right now. If that changes in the future, we can definitely update this.